elasticsearch/templates/statefulset.yaml

```diff
@@ -104,6 +104,7 @@ spec:
       imagePullSecrets:
 {{ toYaml .Values.imagePullSecrets | indent 8 }}
       {{- end }}
+{{- if .Values.sysctlInitContainer.enabled }}
```

Comment:
This will need to go after the `initContainers:` block otherwise this will break `extraInitContainers`. There is a template test that catches this use case too which is failing:

```
________________________________ test_adding_a_extra_init_container _________________________________

    def test_adding_a_extra_init_container():
        config = '''
    extraInitContainers: |
      - name: do-something
        image: busybox
        command: ['do', 'something']
    '''
        r = helm_template(config)
>       extraInitContainer = r['statefulset'][uname]['spec']['template']['spec']['initContainers']
E       KeyError: 'initContainers'
```